src/main/java/org/truffleruby/core/string/StringNodes.java

```diff
@@ -183,10 +183,6 @@ public DynamicObject fromRope(Rope rope) {
             return executeMake(rope, NotProvided.INSTANCE, NotProvided.INSTANCE);
         }
 
-        public DynamicObject fromBuilder(RopeBuilder builder, CodeRange codeRange) {
```

Comment:
I'd like to keep this one for now. I have it there so people don't try to solve this problem their own way. It's a complement to the `fromRope` method above it.

Reply:
ok, will leave it. Please try to make code like that being used at least from a test if possible.